server/src/api/v2/organizations/update-organization.ts

```diff
@@ -0,0 +1,47 @@
+import { Request, Response } from 'express';
+import { checkKeysAreNotEmptyOrNotSet } from '../../../functions/check-inputs.func';
+import { wrapResponse } from '../../../functions/response-wrapper';
+import { RawOrganizationData } from '../../../interfaces/organization.interface';
+import { Organization } from '../../../models/organization.model';
+
+export async function updateOrganization(req: Request, res: Response): Promise<Response> {
+    let success = true;
+    const incomingData: RawOrganizationData = req.body;
+    const organizationId = req.params.id;
+
+    const organizationData: Organization | null = await Organization.findOne(
+        {
+            where: {
+                id: organizationId
```

Comment:
Falls du nochmehr aufräumen willst, gäbe es da noch `findByPk` 😅
https://sequelize.org/master/manual/model-querying-finders.html

Aber ist fraglich, ob das sinnvoll ist, falls später noch Logik eingebaut werden soll (evtl könnte man ja Scope + findByPk machen).

Reply:
Guter Punkt.
Kommt auch auf meine ToDo ;)